reverb/cc/platform/default/build_rules.bzl

```diff
@@ -107,7 +107,7 @@ def reverb_cc_proto_library(name, srcs = [], deps = [], **kwargs):
         name = "{}_static".format(name),
         srcs = gen_srcs,
         hdrs = gen_hdrs,
-        deps = depset(deps + reverb_tf_deps()),
+        deps = depset([dep.replace(":", ":lib") + ".so" for dep in deps] + reverb_tf_deps()),
```

Comment:
can you explain what's going on here?

Reply:
It fixes https://github.com/deepmind/reverb/pull/24#issuecomment-737989183